### DISCUSSION TITLE
v0.13.0

### DISCUSSION THREAD

**Discussion by ledermann** [solectrus/solectrus#2267] [Updates]
**Read the blog post about this release (in German):**
https://solectrus.de/blog/2023-08-13-version-0-13/

## New features

- Add registration (#2265)

## Changes

- Use rounded corners for bar charts (#2213)

## Bug Fixes

- Fixed diagram not continuing when switching between values in NOW view (#2205, #2226)
- Avoid battery level dropping to 0% in NOW view (#2238)

## Internal Changes

- Updated dependencies, including Rails 7.0.7, Chart.JS 4.3.3, and others

**Full Changelog**: [https://github.com/solectrus/solectrus/compare/v0.12.0...v0.13.0](https://github.com/solectrus/solectrus/compare/v0.12.0...v0.13.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/solectrus/solectrus/releases/tag/v0.13.0'>v0.13.0</a>.</em>

**Comment by struppie87** [comment]
Hallo, ich habe nach dem Update das Problem das mir nur der aktuelle Verlauf angezeigt wird. wenn ich etwas anderes auswähle kommt "Serverfehler 500", liegt das an meinem Senechspeicher (Version 1824) oder mache ich etwas falsch?
Der Fehler kommt auch wenn ich mich registrieren möchte.

Danke für die Hilfe
